prow/cluster/jobs/istio-private/proxy/istio-private.proxy.master.yaml

```diff
@@ -0,0 +1,211 @@
+# THIS FILE IS AUTOGENERATED. DO NOT EDIT. See genjobs/README.md
+postsubmits:
+  istio-private/proxy:
+  - branches:
+    - ^master$
+    clone_uri: git@github.com:istio-private/proxy.git
+    cluster: private
+    context: prow/proxy-postsubmit.sh
+    decorate: true
+    decoration_config:
+      gcs_configuration:
+        bucket: istio-private-build
+      ssh_key_secrets:
+      - ssh-key-secret
+    extra_refs:
+    - base_ref: master
+      clone_uri: git@github.com:istio-private/istio.git
+      org: istio-private
+      path_alias: istio.io/istio
+      repo: istio
+    labels:
+      preset-service-account: "true"
```

Comment:
Not needed for this PR but if we're generating the jobs please do not use presets.

Reply:
> Not needed for this PR

Meaning this job does not need a service account? AFAIK it does [activate a SA](https://storage.googleapis.com/istio-prow/logs/proxy-postsubmit/847/build-log.txt) and [uploads build artifacts to to GCS](https://github.com/istio/proxy/blob/c5b7d3418847103059acd63858266d769003f522/prow/proxy-postsubmit.sh#L40)

> if we're generating the jobs please do not use presets.

It would still be nice to keep these as human-readable as possible and not perform too many transformations. Would you recommend expand the `labels` in the job spec?

Reply:
It is more explicit, readable and encapsulated in exchange for being more verbose. The verbosity is not a problem given that it is generated.